.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] Great job on adding the `node_modules` folder to the .gitignone file. however, the folder is still visible in the repository, the reason for this is that you created the file and committed it before adding it to the .gitigone file. to undo this you can run this command `node_modules`. then you can add and commit. with that the folder will not be available in the repository.